content/docs/pulumi-cloud/deployments/api.md

```diff
@@ -1180,7 +1180,7 @@ The `token` is a string that contains the `job`, `offset` and `step` of the next
 
 The following query parameters are available:
 
-* **nextToken**: A string returned by the previous response, that can be used to get the next set of logs.
+* **continuationToken**: A string returned by the previous response, that can be used to get the next set of logs.
```

Comment:
Can we change this to say that it's the `nextToken` value returned by the previous response?

Reply:
Yes, that would be a very helpful clarification.

Reply:
LGTM!